templates/template-publish/content/src/SkorubaIdentityServer4Admin.STS.Identity/Quickstart/Account/RedirectViewModel.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
```

Comment:
STS project uses the Quickstart.UI from https://github.com/IdentityServer/IdentityServer4.Quickstart.UI
All files from this repo have this header with licence information, but this is great notice because I did some small correction in some of these files, I will add notice something like - "Modified by Jan Škoruba", for better clarification, but I think this should be fine.
What do you think?
Thanks.

Reply:
I have added this information in last commit - can you check it please? Is it more clear now?

Reply:
Thank you @TomasHubelbauer , it is really important - please take a look at: https://github.com/skoruba/IdentityServer4.Admin/blob/dev/README.md#licence - I've updated licence section.

Reply:
Yeah, this looks good! :-)